### PR TITLE
CEM-1186 MultiSelect conversion from js to tsx

### PR DESCRIPTION
**Pull request by williancozzi**
Basic:
image

With Columns:
image

### CODE REVIEW

File: stories/Inputs/Multiselect.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export default {
+    title: createStoryTitle('Multi Select'),
+    component: MultiSelect,
+    args: {
+        name: 'Demo',
+    },
+} as Meta;
+
+export const Basic: Story<MultiSelectItemProps> = (args) => (
+    <MultiSelect label={'label'} description={'Description'}>
```
Comment (ralph-dev) [inline stories/Inputs/Multiselect.stories.tsx:20]: Why aren't these in args?

**Review by ralph-dev — APPROVED** [review]
Fix changes before merging

File: stories/Inputs/Multiselect.stories.tsx (hunk 1 above)
Comment (williancozzi) [reply stories/Inputs/Multiselect.stories.tsx]: I have fixed it, but when I pushed, went two more commits with it, that I didnt do it. Is that ok to merge with them?

**Review by ralph-dev — APPROVED** [review]
Lgtm, @Ampatte2 is this ok to merge? I think he brought your landing in after merging master?

**Comment by Ampatte2** [comment]
Repo works fine. I switched the logo picture for Facebook img url since the atlassian link expires.
CEM-1186-Landing-Photo

**Review by ralph-dev — APPROVED** [review]
LGTM

**Comment by Ampatte2** [comment]
LGTM

**Comment by williancozzi** [comment]
Ok, done with Andrew the conflict solution.

**Pull request merged**